crates/solidity/testing/snapshots/bindings_output/yul/external_variables/generated/0.5.8-failure.txt

```diff
@@ -0,0 +1,61 @@
+# This file is generated automatically by infrastructure scripts. Please don't edit by hand.
+
+References and definitions: 
+   ╭─[input.sol:1:1]
+   │
+ 1 │ contract Test {
+   │          ──┬─  
+   │            ╰─── name: 1
+ 2 │     uint d;
+   │          ┬  
+   │          ╰── name: 2
+ 3 │     function test(uint a) public returns (uint b) {
+   │              ──┬─      ┬                       ┬  
+   │                ╰────────────────────────────────── name: 3
+   │                        │                       │  
+   │                        ╰────────────────────────── name: 4
+   │                                                │  
+   │                                                ╰── name: 5
+ 4 │         uint c;
+   │              ┬  
+   │              ╰── name: 6
+   │ 
+ 6 │             c := a
+   │             ┬    ┬  
+   │             ╰─────── ref: 6
+   │                  │  
+   │                  ╰── ref: 4
+ 7 │             b := c
+   │             ┬    ┬  
+   │             ╰─────── ref: 5
+   │                  │  
+   │                  ╰── ref: 6
+ 8 │             c := d.slot
```

Comment:
I think this is a bug? because of the change in `YulIdentifier` grammar. `slot` is not bound here.

Reply:
No, this is expected. `YulIdentifier`s may contain dots specifically from 0.5.8 until 0.7.0. So in this case, `d.slot` is being parsed as a single identifier, not a path.

Before 0.7.0, the syntax to access the slot of a storage variable was to use the `_slot` suffix, eg. `d_slot` in this case.

Reply:
I see. Thanks for explaining.